apps/fluent-tester/src/FluentTester/TestComponents/Notification/NotificationTest.tsx

```diff
@@ -14,10 +18,88 @@ const toastShowDuration = 600;
 const barShowDuration = 300;
 const hideDuration = 250;
 const toastBounciness = 1.5;
-
 const useNativeDriverIOS = true;
 
-const PrimaryTestWithAutoHide: React.FunctionComponent = () => {
+const CustomToast: React.FunctionComponent = () => {
+  const [variant, setVariant] = React.useState<NotificationVariant>('primary');
+  const onVariantChange = React.useCallback((value) => setVariant(value), []);
+
+  const [title, setTitle] = React.useState("Kat's iPhoneX");
+  const [message, setMessage] = React.useState('Listen to Emails • 7 mins');
+  const [action, setAction] = React.useState('Listen');
+
+  const [showIcon, setShowIcon] = React.useState(true);
+  const [showTitle, setShowTitle] = React.useState(true);
+
+  return (
+    <View>
+      <StyledPicker
+        prompt="Variant"
+        selected={variant}
+        onChange={onVariantChange}
+        collection={NotificationVariants.filter((variant) => variant.length < 10)}
+      />
+      <TextInput value={title} onChangeText={setTitle} style={{ borderWidth: 1 }} />
```

Comment:
Nit: put style inside a Stylesheet so it's (maybe?) not recreated on every render